.eslintrc

```diff
@@ -685,6 +691,30 @@
           }
         ]
       }
+    },
+    {
+      "files": [
+        "packages/story-editor/src/**/*.js",
+        "packages/dashboard/src/**/*.js",
+        "packages/design-system/src/**/*.js",
+        "packages/element-library/src/**/*.js"
+      ],
```

Comment:
This probably affects a bunch more packages.

Might be easiest to do `"files": [ "packages/**/*.js" ],` and then add all non-editor packages (e.g. widget, lightbox, etc.) to `excludedFiles`

Reply:
You are right. 
These are all the `@web-stories-wp` packages.

- e2e-tests
- activation-notice
- commander
- e2e-test-utils
- eslint-import-resolver
- glider
- imgareaselect
- jest-amp
- jest-puppeteer-amp
- jest-resolver
- karma-failed-tests-reporter
- karma-puppeteer-client
- karma-puppeteer-launcher
- stories-block
- stories-carousel
- stories-lightbox
- tinymce-button
- widget
- wp-dashboard
- wp-story-editor
- wp-utils

But not all have React as a dependency, so they would not be using `useEffect` anyways. These are the `@web-stories-wp` packages with React dependency, so added those in the `excludedFiles`.

- activation-notice
- jest-amp
- karma-fixture
- react
- stories-block
- test-utils
- tinymce-button
- wp-dashboard
- wp-story-editor
- wp-utils